File: src/Ingredient.java
public class Ingredient {

     boolean isArchived;
     String name;
     int categorie;
     double caloriesPerGram;
     double pricePerKilo;

    public Ingredient(boolean isArchived, String name, int categorie, double caloriesPerGram, double pricePerKilo) {
        this.isArchived = isArchived;
        this.name = name;
        this.categorie = categorie;
        this.caloriesPerGram = caloriesPerGram;
        this.pricePerKilo = pricePerKilo;
    }
}

File: src/classes/Ingredient.js
export class Ingredient {
  constructor(name, category, caloriesPerGram, pricePerKilo) {
    this.isArchived = false;
    this.name = name;
    this.category = category;
    this.caloriesPerGram = caloriesPerGram;
    this.pricePerKilo = pricePerKilo;
  }

  static createNew() {
    return new Ingredient('', 0, 0, 0);
  }
}
